### DISCUSSION TITLE
Video playback flickering on iOS 15.11

### DISCUSSION THREAD

**Discussion by andreasx23** [BattlesnakeOfficial/feedback#147] [Bug Reports]
I noticed when doing video playback of a game while it’s running opening the settings window and closing it again will course the video playback to start flicking between two matches

iOS 15.11
iPhone 12
Safari 15.1 browser

**Comment by originalwebgurl** [comment]
Thanks for the report @andreasx23. I'll investigate and get back to you. We currently don't support Opera mini, do you know if it's Opera mini or Opera mobile that you're using, and what version? This link may help: https://www.whatismybrowser.com/detect/what-version-of-opera-do-i-have or you can check in the Opera About menu item to grab the version info.

**Reply by andreasx23** [reply]
Oh my silly me 😅 was so tired when I wrote this bug report that I mistakenly said Opera when I meant Safari. I’m using Safari 15.1

**Comment by EbbDrop** [comment]
This seems to also happen on desktop if you have autoplay enabled.
